app/exporters/formatters/countryside_stewardship_grant_indexable_formatter.rb

```diff
@@ -7,7 +7,12 @@ def type
 
   private
   def extra_attributes
-    {}
+    {
+      grant_type: entity.grant_type,
+      land_use: entity.land_use,
+      tiers_or_standalone_items: entity.tiers_or_standalone_items,
+      funding_per_unit_per_year: entity.funding_per_unit_per_year,
```

Comment:
I was thinking that myself but I thought I'd put in the full label for the sake of discussing it. I'll change it now.